sphinx_search/_static/js/rtd_sphinx_search.js

```diff
@@ -521,12 +588,66 @@ window.addEventListener("DOMContentLoaded", evt => {
             removeSearchModal();
         });
 
+        // copy the url, to clipboard, to the search
+        // UI when the button is clicked.
+        linkToSearchButton.addEventListener("click", e => {
+            e.preventDefault();
+
+            const host = window.location.origin;
+            const path = window.location.pathname;
+            let urlToBeCopied = host + path;
+
+            if (
+                SEARCH_QUERY !== undefined &&
+                SEARCH_QUERY !== null &&
+                SEARCH_QUERY !== ""
+            ) {
+                let urlParam = convertObjToUrlParams({
+                    rtd_search: encodeURIComponent(SEARCH_QUERY)
+                });
+                urlToBeCopied += "?" + urlParam;
+            }
+
+            copyToClipBoard(urlToBeCopied);
```

Comment:
I don't quite follow what this is doing. We should just update the browser's URL when the user types with the proper data, we don't need a custom way to copy a link.

Reply:
This is generating the url to the search ui and then copying it to the clipboard.